### PR TITLE
CAL-240 updated changelog

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates changelog for 0.2.1 release

#### Who is reviewing it?

@mcalcote @Lambeaux @jrnorth @clockard 

#### Choose 2 committers to review/merge the PR.
@coyotesqrl
@figliold
@jaymcnallie
@jlcsmith
@lessarderic
@pklinef
@shaundmorris
@stustison
#### How should this be tested?

preview changes at https://github.com/ricklarsen/alliance/blob/CAL-240/CHANGES.md

#### Any background context you want to provide?

Seeking suggestions for additional content. Leave as a comment on this PR.

#### What are the relevant tickets?

[CAL-240](https://codice.atlassian.net/browse/CAL-240)

#### Screenshots (if appropriate)

n/a

#### Checklist:
- [ ] Documentation Updated
	- [x] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -32,29 +32,50 @@ replace with next unreleased version
...
  -->
 
-## 0.2.1
+## 0.3.0
 	Release Date: pending
 
 ### NOTES
 
-- Built on [DDF v.2.10.1](https://github.com/codice/ddf/blob/master/CHANGES.md#2101)
+- Built on [DDF v.2.11.1](https://github.com/codice/ddf/blob/master/CHANGES.md#2110)
```
Comment (coyotesqrl) [inline CHANGES.md]: More likely 0.3.0 will be built on 2.11.0. It's all just guesswork at this point, though.

**Comment by ricklarsen** [comment]
@coyotesqrl Good catch. Fixed.

File: CHANGES.md [hunk 2]
```diff
@@ -32,29 +32,50 @@ replace with next unreleased version
...
+
+## 0.2.1
+	Release Date: 2017-03-08
+
+### NOTES
+
+- Built on [DDF v.2.10.1](https://github.com/codice/ddf/blob/master/CHANGES.md#2101)
+
+### NEW FEATURES
+
```
Comment (mcalcote) [inline CHANGES.md]: CAL-258 Extract NitfPreStoragePlugin into its own bundle/feature (#259) [Chris Lockard]

File: CHANGES.md [hunk 3]
```diff
@@ -1,175 +1,507 @@
...
+	<li><a href='https://codice.atlassian.net/browse/CAL-257'>CAL-257</a> - Clean up maven profiles and add static analysis deactivation
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/CAL-260'>CAL-260</a> - Upgrade to latest version of Yarn
+	</li>
+</ul>
 
 ## 0.2.0
 	Release Date: 2017-02-08
+Built with [DDF Version 2.10.0](https://github.com/codice/ddf/blob/master/CHANGES.md#2100)
+<h3>Sub-task</h3>
```
Comment (lessarderic) [inline CHANGES.md]: ❗️ Remove?

**Pull request merged**